packages/mobile/src/geth/network-config.ts

```diff
@@ -4,26 +4,31 @@ import { Testnets } from 'src/web3/testnets'
 export default {
   [Testnets.integration]: {
     nodeDir: `.${Testnets.integration}`,
+    noDiscovery: true,
```

Comment:
The wording here seems a bit awkward. can this be reversed to `discovery: false`?